### PR TITLE
Created diff eqs

### PR DESCRIPTION
**Pull request by stefan-wass**

### CODE REVIEW

File: rk4_solver.py [hunk 1]
```diff
@@ -3,41 +3,100 @@
...
+    float
+        The change in efffector cells after one timestep
+    """
+    s = 1.2 * 10**4
+    mu = 4.12 * 10**(-2)
+    p = 0.015
+    h = 20.2
+    m = 2 * 10**(-11)
+    K_e = 1 # not sure what this is yet
+    dE = s-(mu*E) + p*((E*T)/(h+T)) - (m*E*T) - (K_e*M*E)
```
Comment (SidhantPuntambekar) [inline rk4_solver.py:76]: shouldn't you add p(ET/h +t) before subtracting mu*E?

**Review by SidhantPuntambekar — CHANGES_REQUESTED** [review]
It's interesting that all of these differential equations are multivariate so our Runge-Kutta solver should theoretically adapt to handle the variables of these equations. I'll need to make changes to the rk4 function to accommodate I think.

**Comment by SidhantPuntambekar** [comment]
Also clean out all of the TODO items in the docstrings here

File: rk4_solver.py (hunk 1 above)
Comment (SidhantPuntambekar) [reply rk4_solver.py:76]: Following discussion with Stefan, this equation is good. Lestari et al. are just bad at keeping their equations formatted correctly.

**Review by SidhantPuntambekar — APPROVED** [review]
Looks good to merge

**Comment by ssarthak01** [comment]
looks good to me

**Review by ssarthak01 — APPROVED** [review]
looks good!

**Pull request merged**